src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -1,5 +1,12 @@
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 import disabilityLabelsReduced from './disabilityLabelsReduced';
+import disabilityLabelsRevised from './disabilityLabelsRevised';
+import revisedFormWrapper from './revisedFormWrapper';
 
 export const getDisabilityLabels = () => {
+  if (revisedFormWrapper.isRevisedForm(environment.isStaging())) {
```

Comment:
lgtm!

wondering for future projects, if it would be helpful to use `!environment.isProduction()` rather than `environment.isStaging()`. this would allow the changes to work in all lower environments (e.g. local) for manual qa

Reply:
that's a great idea, I'll employ that next time, if I need to use an environment flag!